api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -539,7 +539,7 @@ public List<ReportingWorkspace> getWorkspaceBatch(long limit, long offset) {
             + "  LEFT OUTER JOIN featured_workspace fw ON w.workspace_id = fw.workspace_id\n"
             + "  LEFT OUTER JOIN user_initial_credits_expiration uice ON uice.user_id = creator_id\n"
             + "LEFT JOIN user_verified_institutional_affiliation uvia ON w.creator_id = uvia.user_id\n"
-            + "JOIN institution i ON uvia.institution_id = i.institution_id\n"
```

Comment:
This was causing count mismatch failures on Test, because some workspace creators didn't have institutional affiliations.

This query is unchanged in semantics because it already accounts for lack of affiliation on L496.